embrace-gradle-plugin-integration-tests/src/test/java/io/embrace/android/gradle/integration/testcases/AndroidNdkTest.kt

```diff
@@ -17,7 +17,8 @@ class AndroidNdkTest {
     @JvmField
     val rule: PluginIntegrationTestRule = PluginIntegrationTestRule()
 
-    private val defaultExpectedVariants = listOf("debug", "release")
+    private val defaultExpectedVariants = listOf("release")
+    private val variantsSentInBuildTelemetry = listOf("debug", "release")
```

Comment:
Do we want to send build telemetry for debug builds? I would argue not

Reply:
If we're not uploading mapping files for debug builds then I don't see a reason to capture telemetry, as long as we do capture it for release builds & end-users can opt-out of collection